src/dst/src/Distributed.cc

```diff
@@ -104,6 +106,32 @@ void Distributed::runLoadBalancer(const char* ip, unsigned short port)
   }
 }
 
+bool Distributed::lookUpWorkers(const char* DNS, unsigned short port)
+{
+  asio::io_service ios;
+  asio::ip::udp::resolver::query resolver_query(DNS, std::to_string(port), asio::ip::udp::resolver::query::numeric_service);
```

Comment:
too many namespaces. replace with using asio::ip::udp and using asio::ip::udp::resolver